private/bufpkg/bufconnect/interceptors_test.go

```diff
@@ -25,14 +25,32 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestAuthorizationInterceptorProviderTokenErr(t *testing.T) {
-	_, err := NewAuthorizationInterceptorProvider(app.NewEnvContainer(map[string]string{
-		tokenEnvKey: "test1234",
-	}))("fake")(func(context.Context, connect.AnyRequest) (connect.AnyResponse, error) {
-		return nil, errors.New("underlying cause")
+func TestNewAuthorizationInterceptorProvider(t *testing.T) {
```

Comment:
Check for no header when there is no token for the remote.

Reply:
Now it checks the header for `AuthenticationHeader` is empty.